docs/trex_getstarted.md

```diff
@@ -91,13 +91,13 @@ To use the dashboard extension samples, you need to start up a web server on you
 ---
 ### Start Tableau and add an extension to the dashboard
 
-1. Start Tableau and open a workbook that has a dashboard, or open a workbook and create a new dashboard.
+1. Start Tableau and open a workbook that has a dashboard, or open a workbook and create a new dashboard with at least one worksheet.
```

Comment:
Why are we saying it needs to have a worksheet?

Reply:
@bcantoni had that comment/change in his PR, so I kept it. On the other hand, we tell them to use the Datasources sample, so we should probably say open a Superstore workbook, or a workbook that has a data source.

Reply:
OK. I changed this to say: 

"Start Tableau and open a workbook that has a dashboard, or open a workbook and create a new dashboard. For example, you could use one of the Tableau sample workbooks like Superstore to start with."

Reply:
I think this wording sounds good. My original "with at least one worksheet" didn't make much sense, but what I intended here was to suggest they use a dashboard with something on it. We instruct people to use the Samples/DataSources.trex extension, but it won't have anything to report if used on a new empty dashboard.